indico/MaKaC/webinterface/mail.py

```diff
@@ -162,7 +162,7 @@ def send( notification, fromAddress="" ):
         maildata = {
             "fromAddr": fromAddr,
             "toList": [toAddr],
-            "subject": "[Indico] %s" % notification.getSubject(),
+            "subject": _("[Indico] %s") % notification.getSubject(),
```

Comment:
The name "Indico" is not translatable. Also, we don't translate emails at the moment since they'd need to use the recipient's language (which is not supported in the current mailing system).